examples/com.eclipsesource.modelserver.example/src/main/java/com/eclipsesource/modelserver/example/ExampleServerLauncher.java

```diff
@@ -35,23 +39,37 @@
 
 	private static Logger LOG = Logger.getLogger(ExampleServerLauncher.class);
 
-	public static void main(String[] args) {
+	public static void main(String[] args) throws ParseException {
 		BasicConfigurator.configure();
+		CLIParser.create(args, CLIParser.getDefaultCLIOptions());
+		Logger root = Logger.getRootLogger();
+		if (CLIParser.getInstance().optionExists("e")) {
```

Comment:
Reading the CLI options seems as if it should be part of `ModelServerLauncher` (as we do with the port). Is there a reason why this is located here?

Reply:
Technically you are right (at least the CLI option for logging can be moved into the ModelServerLauncher).

We still need to read the CLI options here to check the workspace root. If no root was specified we use the testworkspace as a fallback (and modify the launch args accordingly)

Reply:
Moving the logLevel CI-check to the `ModelServerLauncher` doesn't work properly. Some debug logs slip through.

Maybe we should leave this as is for the moment. I will create an issue for improving the log configuration (i.e Errors are tracked exclusively to the error stream). Once this issue is resolved the cli loglevel configuration could be removed entirely.